src/Designer/frontend/admin/features/apps/components/AppsTable.tsx

```
@@ -61,34 +67,120 @@ const AppsTableWithData = ({ runningApps }: AppsTableWithDataProps) => {
+            </StudioTable.Cell>
+          </StudioTable.Row>
+        </StudioTable.Head>
+        <StudioTable.Body>
+          {runningApps[env]
+            .filter((app) => !search || app.app.toLowerCase().includes(search.toLowerCase()))
+            .map((app) => {
+              const appErrorMetrics = errorMetrics?.filter((metric) => metric.appName === app.app);
+              return {
+                ...app,
+                metrics: appErrorMetrics,
+                hasMetrics: appErrorMetrics?.length ?? 0 > 0,
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)